ext/node/crypto/mod.rs

```diff
@@ -153,3 +154,33 @@ pub fn op_node_public_encrypt(
     _ => Err(type_error("Unknown padding")),
   }
 }
+
+#[op(fast)]
+pub fn op_node_create_cipheriv(
+  state: &mut OpState,
+  algorithm: &str,
+  key: &[u8],
+  iv: &[u8],
+) -> u32 {
+  state.resource_table.add(
+    match cipher::CipherContext::new(algorithm, key, iv) {
+      Ok(context) => context,
+      Err(_) => return 0,
+    },
+  )
```

Comment:
Appears that this resource will never be dropped which will cause the same issue as here: https://github.com/denoland/deno/issues/18140

Reply:
Not sure this is the correct way to implement it, but `Cipheriv.prototype.final` method calls `Deno.close(this.context)` which closes the cipher resource, and the test cases pass without disabling the sanitizer